client/src/pages/customer-pages/Home/index.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from "react";
+import SweetAlert from "react-bootstrap-sweetalert";
+
+import {
+  Input,
+  Button,
+  CustomerPageNavBar,
+  Checkbox
+} from "../../../components";
+import "./home.css";
+
+class Home extends Component {
+  state = {
+    textInputValueFlightNo: "",
+    checkboxValue: true,
+    isAvailable: false,
+    alert: null
+  };
+
+  handleNoInputChange = e => {
+    this.setState({
+      textInputValueFlightNo: e.target.value
+    });
+  };
+
+  handlecheckboxChange = () => {
+    this.setState({
+      checkboxValue: !this.state.checkboxValue
+    });
+  };
+
+  invalidFlightNoAlert = () => {
+    const getAlert = () => (
+      <SweetAlert
+        warning
+        title="Wrong!"
+        confirmBtnBsStyle="danger"
+        cancelBtnBsStyle="danger"
+        onConfirm={() => this.hideAlert()}
+      >
+        Invalied Flight no. !
+      </SweetAlert>
+    );
+
+    this.setState({
```

Comment:
Here, you can store a flag in this.state which says whether to show the SweetAlert component, and then in render(), read that flag and put the SweetAlert component right there in render(), if the flag is true.  That way, this.state just has simple (and serializable) data types in it.